script.js

```diff
@@ -0,0 +1,32 @@
+const rock = "rock";
+const paper = "paper";
+const scissors = "scissors";
+const compVal = ['rock','paper','scissors'];
+let humanVal;
```

Comment:
This is not used anywhere, it needs to be removed

Reply:
fixed